documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -165,3 +165,42 @@ spec:
           key: cruise-control-log4j.properties
     # ...
 ----
+
+[[api-security-configuration]]
+[discrete]
+== API security
+
+The Cruise Control API is secured by default to protect the cluster against any potentially destructive Cruise Control operations.
+
+However, these security settings can be disabled.
+
```

Comment:
_Add a bulleted list:_

* To disable the built-in authorization, set `webserver.security.enable` to `false`.
* To disable the built-in authentication, set `webserver.ssl.enable` `false`.

Reply:
Resolved.